js/main.js

```diff
@@ -0,0 +1,25 @@
+const $photoURL = document.querySelector('#url');
+const $form = document.querySelector('form');
+
+$photoURL.addEventListener('input', event => {
+  const url = document.querySelector('#url').value;
```

Comment:
Instead of doing another `querySelector` inside this function, I would either utilize the `event.target.value` or your `$photoURL` variable. This event is going to run on every key press, so we don't want to traverse through the DOM every-time this event fires.